google/cloud/forseti/services/inventory/base/cloudasset.py

```diff
@@ -180,13 +180,20 @@ def _export_assets(cloudasset_client, config, root_id, content_type):
         if asset_types:
             LOGGER.info('Limiting export to the following asset types: %s',
                         asset_types)
-
-        results = cloudasset_client.export_assets(root_id,
-                                                  export_path,
-                                                  content_type=content_type,
-                                                  asset_types=asset_types,
-                                                  blocking=True,
-                                                  timeout=timeout)
+        try:
+            results = cloudasset_client.export_assets(root_id,
+                                                      export_path,
+                                                      content_type=content_type,
+                                                      asset_types=asset_types,
+                                                      blocking=True,
+                                                      timeout=timeout)
+        except api_errors.HTTPError as e:
```

Comment:
I think the lower level `export_assets` method would raise the error here as `api_errors.ApiExecutionError`. So, that should be what you catch, and then check on the `400` resp.code

Reply:
Addressed.